ibu-imager/seedcreator/seedcreator.go

```diff
@@ -222,6 +229,73 @@ func (s *SeedCreator) stopServices() error {
 	return nil
 }
 
+func (s *SeedCreator) runRecertValidation() error {
```

Comment:
I think we need a way to disable this validation, we can have default "run=true" but will be nice to have a way to run without it as i am not sure why i always need it in dev process

Reply:
As a workaround you can pass in a dummy image that doesn't do anything

Reply:
why not to add flag for skip?

Reply:
Not opposed, I said workaround

Reply:
k, added `--skip-recert-validation` flag, feel free to give it a spin